### PR TITLE
Grails upgrade preparation

### PR DESCRIPTION
**Pull request by klodav**

### CODE REVIEW

File: app/grails-app/views/templates/cpa/_person_table.gsp [hunk 1]
```diff
@@ -71,7 +71,7 @@
                     }
                 %>
                 <div class="ui divided middle aligned list la-flex-list ">
-                    <g:each in="${pRolesSorted.sort{it.functionType ? it.functionType.getI10n('value') : it.positionType.getI10n('value')}}" var="role">
+                    <g:each in="${pRolesSorted.sort{it.functionType ? it.functionType?.getI10n('value') : it.positionType?.getI10n('value')}}" var="role">
```
Comment (croqueGrec09) [inline app/grails-app/views/templates/cpa/_person_table.gsp:74]: why null checks?

**Review by croqueGrec09 — COMMENTED** [review]
one question

File: app/grails-app/views/templates/cpa/_person_table.gsp (hunk 1 above)
Comment (klodav) [reply app/grails-app/views/templates/cpa/_person_table.gsp:74]: think about it

Comment (croqueGrec09) [reply app/grails-app/views/templates/cpa/_person_table.gsp:74]: Jira-issue raised

**Review by croqueGrec09 — APPROVED** [review]
approved

**Pull request merged**